lib/tulul_stats_bot/tulul_stats_bot.rb

```diff
@@ -11,7 +11,11 @@ def self.start
 
           query = /\/top_(.+)/.match(message.text).captures[0] rescue nil
           query = query.split('@')[0] rescue nil
-          if query && (TululStats::User.fields.keys.reject{ |field| TululStats::User::EXCEPTION.include?(field) } + TululStats::Entity::ENTITY_QUERY + TululStats::IsTime::TIME_QUERY).include?(query)
+          if /^\/last_tulul([@].+)?/.match(message.text.strip)
```

Comment:
we can go with this or use `/\/([top|last]_.+)/` and pass the whole command to `Group#top`